app/controllers/accounts_controller.rb

```diff
@@ -0,0 +1,33 @@
+class AccountsController < ApplicationController
+  before_action :load_account
+
+  def edit
+  end
+
+  def update
+    @account.balance = @account.balance + account_params[:balance].to_f
```

Comment:
@babarajay , Is there any way to not use instance variable for `@account`?

Reply:
@babarajay ,Why there is no transaction entry for deposit? Can you rectify this?

Reply:
@babarajay , What if a balance is negative?

Reply:
Is there any way to not use instance variable for @account?
-> I've used @account instance variable to initialize form. I can use current_user.account instead.

Why there is no transaction entry for deposit? Can you rectify this?
-> I'll resolve it.

What if a balance is negative?
-> I'll resolve it.